### PR TITLE
updated-lorem2

### PR DESCRIPTION
**Pull request by draganjitmedia**
change of lorem text inside some div

### CODE REVIEW

File: index1.html [hunk 1]
```diff
@@ -1,314 +0,0 @@
...
-                <div class="column2" class="info2">
-                    <img class="50%" src="images/nike.png" alt="">
-                    <h4 class="artikal">Artikal 6</h4>
-                    <h3 class="cijena">300 <span style="color: black;">150</span></h3>
-                    <p><button class="buttonkupime">Kupi me</button></p>
-                </div>
-            </div>
-        </div>
-        <!-- <input name="slide" type="radio" value="" checked>///////////////////sacekati kasnije///////////////////////////////////
-        <input name="slide" type="radio" value="">
```
Comment (zlayabekrija) [inline index1.html]: Please remove commented line

File: index1.html [hunk 2]
```diff
@@ -1,314 +0,0 @@
...
-                <p>Terry Couper</p>
-                <h6 class="mala-slova">Lorem ipsum dolor sit amet consectetur adipisicing elit. Error magni saepe sed
-                    voluptate sint. Sunt co mmodi beatae e expedita dolorum natus aspernatur, sit modi voluptatem quia?
-                </h6>
-
-            </div>
-        </div>
-    </section>
-
-    <section class="">
```
Comment (zlayabekrija) [inline index1.html]: We shouldn't use empty class names

File: index1.html [hunk 3]
```diff
@@ -1,314 +0,0 @@
...
-        <div class="bg-gradient">
-            <h2 class="text-white text-center">What 60seconds members are getting</h2>
-            <hr>
-            <div class="row" id="bg-gradient">
```
Comment (zlayabekrija) [inline index1.html]: is Id bg-gradient descriptive enough. We should use id to distinguish parts of the code

File: index1.html [hunk 4]
```diff
@@ -1,314 +0,0 @@
...
-                </div>
-                <div class="column" class="info">
-                    <img src="images/lotto-logo.png" alt="">
-                    <h4 class="artikal">Socks</h4>
-                    <h3 class="cijena">40 <h5>$15</h5>
-                    </h3>
-                    <p><button class="buttonkupime">Shop Now</button></p>
-                </div>
-            </div>
-            <!--ekperiment ispod////////////////////////////////////////////////////////////////////////////////////////////-->
```
Comment (zlayabekrija) [inline index1.html]: English only please

File: index1.html [hunk 5]
```diff
@@ -1,314 +0,0 @@
...
-            </iframe>
-        </div>
-    </section>
-    <section class="artikli">
-
-        <h2>So many great deals...</h2>
-        <p>The customer is the game-changer with just one click selects the virtual store of the brand , the product
-            that they want to buy , and he price that they want to pay.</p>
-
-        <div class="row" id="artikli">
```
Comment (zlayabekrija) [inline index1.html]: Please use English naming only, and avoid having same names for classes and ids. Remember that classes are general while ids are specific.

File: index1.html [hunk 6]
```diff
@@ -1,314 +0,0 @@
...
-
-        </section>
-        <section class="flex-item-right">
-            <div>
-                <img src="images/Group 64.png" alt="">
-            </div>
-        </section>
-    </section>
-    <section class="video">
-        <div class="videoplejerhidden">
```
Comment (zlayabekrija) [inline index1.html]: Separate words by kebab case . Ex. video-player-hidden

File: index1.html [hunk 7]
```diff
@@ -1,314 +0,0 @@
...
-                <p>We designed our layout to be simple , no-frills, and easy to navigate. The key is t oresist thinking
-                    too
-                    hard . Relax, and enjoy your shopping experience like never before.</p>
-            </div>
-            <button type="button" class="button" id="lijevo"><a href="#">Get the app</a></button>
-
-        </section>
-        <section class="flex-item-left">
-            <div>
-                <img src="images/Group 76.png" alt="">
```
Comment (zlayabekrija) [inline index1.html]: Please make sure to fill up the alt attributes for images.

File: index1.html [hunk 8]
```diff
@@ -1,314 +0,0 @@
...
-
-        </section>
-        <section class="flex-item-right">
-            <div>
-                <img src="images/Group 77.png" alt="">
-            </div>
-        </section>
-    </section>
-    <section class="flex-container">
-        <section class="deals" class="flex-item-right" id="desno">
```
Comment (zlayabekrija) [inline index1.html]: No language other than English, please.

File: index1.html [hunk 9]
```diff
@@ -1,314 +0,0 @@
...
-                <!-- </div> -->
-                <!-- <div class="row"> -->
-                <div class="column">
-                    <img src="images/nike.png" alt="">
-                </div>
-                <div class="column">
-                    <img src="images/Maserati-logo.png" alt="">
-                </div>
-                <!-- </div> -->
-                <!-- <div class="row"> -->
```
Comment (zlayabekrija) [inline index1.html]: Remove commented code please.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great effort, 
there are some minor changes to be done, and also some things that we need to start applying from the very beginning. Those would be: 

- English only
- Ids only for specific elements 
- No empty attributes
- kebab case for css

Please resolve those issues and resubmit pull request.

**Comment by draganjitmedia** [comment]
please ,, review again for changes.

**Pull request merged**